cognite/client/data_classes/time_series.py

```diff
@@ -48,7 +49,7 @@ class TimeSeriesCore(WriteableCogniteResource["TimeSeriesWrite"], ABC):
         description (str | None): Description of the time series.
         security_categories (Sequence[int] | None): The required security categories to access this time series.
         data_set_id (int | None): The dataSet ID for the item.
-        legacy_name (str | None): Set a value for legacyName to allow applications using API v0.3, v04, v05, and v0.6 to access this time series. The legacy name is the human-readable name for the time series and is mapped to the name field used in API versions 0.3-0.6. The legacyName field value must be unique, and setting this value to an already existing value will return an error. We recommend that you set this field to the same value as externalId.
+        legacy_name (str | None): : This field is not used by the API and will be removed in a future version of the SDK.
```

Comment:
Note the double colon:
```suggestion
        legacy_name (str | None): This field is not used by the API and will be removed in a future version of the SDK.
```